src/Microsoft.VisualBasic.Core/tests/StringsTests.cs

```diff
@@ -189,6 +190,187 @@ public void Filter_Objects(object[] source, string match, string[] includeExpect
             Assert.Equal(excludeExpected, Strings.Filter(source, match, Include: false));
         }
 
+        [Theory]
+        [MemberData(nameof(Format_TestData))]
+        public void Format(object expression, string style, string expected)
+        {
+            Assert.Equal(expected, Strings.Format(expression, style));
+        }
+
+        [Theory]
+        [MemberData(nameof(Format_InvalidCastException_TestData))]
+        public void Format_InvalidCastException(object expression, string style)
+        {
+            Assert.Throws<InvalidCastException>(() => Strings.Format(expression, style));
+        }
+
+        private static IEnumerable<object[]> Format_TestData()
+        {
+            yield return new object[] { null, null, "" };
+            yield return new object[] { null, "", "" };
+            yield return new object[] { "", null, "" };
+            yield return new object[] { "", "", "" };
+            yield return new object[] { sbyte.MinValue, "0", "-128" };
+            yield return new object[] { sbyte.MaxValue, "0", "127" };
+            yield return new object[] { ushort.MinValue, "0", "0" };
+            yield return new object[] { ushort.MaxValue, "0", "65535" };
+            yield return new object[] { false, "", "False" };
+            yield return new object[] { false, "0", "0" };
+            if (IsEnUS())
```

Comment:
Isntead of testing for en-US, why not just set thread culture temporarily to en-US? I'm guessing, but don't know for certain, that all localized editions of Windows contain en-US data, since I assume, but don't know for certain, that it's used as "neutral" culture data.
@tarekgh on es-ES Windows, is en-US culture data always available?

Reply:
It's really not important though. This can be submitted as is.

Reply:
Thanks @danmosemsft, I'll consider switching to that approach in a subsequent PR. (I suspect there will be more test cases where we'll need this and I'll make the change at that time.)

Reply:
For many of these the "simple" API like `String.Format` gives the same result presumably (ie the VB functions are just pass throughs for VB6 compat). In that case you can just compare the result to String.Format, and you don't need to care about the culture. Of course by that argument you also need hardly any tests anyway

Reply:
> on es-ES Windows, is en-US culture data always available

Just to answer that even if it is not needed. the answer is Yes. en-US is available all the time on all OS's with all localized versions.